apps/pie-docs/turbo.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": [
+    "//"
```

Comment:
Does the `//` refers to the root `turbo.json`?

Reply:
Yes indeed! https://turbo.build/repo/docs/core-concepts/monorepos/configuring-workspaces#how-it-works

See the note below the code snippet